cloud/google/clients/computeservice.go

```diff
@@ -1,9 +1,34 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package clients
 
 import (
-	compute "google.golang.org/api/compute/v1"
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/golang/glog"
```

Comment:
Please group standard library packages first, separate from the other packages
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
done